packages/nextjs-template/components/DendronTreeMenu.tsx

```diff
@@ -232,8 +246,18 @@ function MenuItemTitle(props: Partial<NoteData> & { menu: DataNode }) {
         href={getNoteUrl(props.menu.key as string, {
           noteIndex: props.noteIndex!,
```

Comment:
I think maybe `getNoteUrl` should allow for `props.noteIndex` to be undefined and handle it appropriately instead of us having to pass it along from the top and doing a non-null assertion each time.

Reply:
yes, good thing 4d64edb3e